lib/resources/dimensions.dart

```diff
@@ -25,4 +25,6 @@ class Dimensions {
   static const double homeSkeletonLoadingTextBorderRadius = 10.0;
 
   static const double answerEmojiSize = 34.0;
+
+  static const double divider = 1.0;
```

Comment:
```suggestion
  static const double answerNpsDividerWidth = 1.0;
```

Because the divider for another answer use a different value here. 😢